samples/drivers/led_gpio/README.rst

```diff
@@ -0,0 +1,31 @@
+.. _gpio-led-sample:
+
+LED GPIO Sample
+###############
+
+Overview
+********
+
+The sample assumes that the LEDs are connected to GPIO lines.
+It is configured to work on boards that have defined the led0, led1, led2, led3
+aliases in their board devicetree description file.
```

Comment:
Why aliases are required?

Reply:
Aliases are probably more common to use in sample or application. I did change sample to use it.

Reply:
You might need to mention the use of `id` property also, since that is a requirement for the driver.